datafusion/sqllogictest/test_files/functions.slt

```diff
@@ -723,6 +723,36 @@ SELECT split_part(arrow_cast('foo_bar', 'Dictionary(Int32, Utf8)'), '_', 2)
 ----
 bar
 
+query ?
```

Comment:
Can you also please add a test that works on tables (not just scalars).

Something like 

```sql
create table strings(value varchar) as values ('foo'), ('abc def'), ('abc'), NULL;
select string_to-array(value) from values
```

Reply:
Done